spring-boot-project/spring-boot-docs/src/docs/asciidoc/actuator/metrics.adoc

```diff
@@ -429,20 +480,22 @@ By default, metrics are published via REST calls but it is also possible to use
 	      newrelic:
 	        client-provider-type: "insights-agent"
 ----
+====
 
 Finally, you can take full control by defining your own `NewRelicClientProvider` bean.
 
 
 
 [[actuator.metrics.export.prometheus]]
 ==== Prometheus
-{micrometer-registry-docs}/prometheus[Prometheus] expects to scrape or poll individual app instances for metrics.
-Spring Boot provides an actuator endpoint available at `/actuator/prometheus` to present a https://prometheus.io[Prometheus scrape] with the appropriate format.
+{micrometer-registry-docs}/prometheus[Prometheus] expects to scrape or poll individual application instances for metrics.
+Spring Boot provides an available actuator endpoint at `/actuator/prometheus` to present a https://prometheus.io[Prometheus scrape] with the appropriate format.
```

Comment:
It's not the endpoint that's available. "available" was being used to describe the location at which it can be accessed. Perhaps "Spring Boot provides an actuator endpoint, available at `/actuator/prometheus`, to present … " instead?

Reply:
Removing "available" altogether does the trick. I did that.